### PR TITLE
291  unit tests to package mapping

### PR DESCRIPTION
**Pull request by KizerovDmitriy**

### CODE REVIEW

File: service/src/test/java/greencity/mapping/MultipartBase64ImageMapperTest.java
Comment (DenysLiubchenko) [inline service/src/test/java/greencity/mapping/MultipartBase64ImageMapperTest.java:1]: MultipartBase64ImageMapper lines covered 88%. Catch block isn`t covered

**Review by DenysLiubchenko — CHANGES_REQUESTED** [review]

File: service/src/test/java/greencity/mapping/MultipartBase64ImageMapperTest.java
Comment (KizerovDmitriy) [reply service/src/test/java/greencity/mapping/MultipartBase64ImageMapperTest.java:1]: 100% Coverage

**Review by urio999 — APPROVED** [review]
Cool! Great job, guys

**Review by DenysLiubchenko — COMMENTED** [review]
Nice

**Review by DenysLiubchenko — APPROVED** [review]

**Review by staaankey — APPROVED** [review]

**Pull request merged**